Source/PP_Spray/SpraySetup.cpp

```diff
@@ -125,16 +130,59 @@ SprayParticleContainer::readSprayParams(int& particle_verbose)
     }
   }
 
-  Real parcel_size = 1;
   Real spray_ref_T = 300.;
   bool splash_model = false;
+  int breakup_model = 0;
   //
   // Set the number of particles per parcel
   //
-  pp.query("parcel_size", parcel_size);
```

Comment:
The test cases in LMeX and C are set up to have parcel_size of 10 and 100, so this change, which effectively makes parcel_size always 1 without splash/breakup models, leads to different results for those tests. Running the old MP code with a parcel_size of 1 results in no diffs vs. this code. Just want to make sure that this is the correct anticipated behavior after this PR, and we can simply remove the parcel_size input file entries going forward.

https://github.com/AMReX-Combustion/PeleLMeX/blob/56740f63b9ed9650ceb4cae171e6e561bc983431/Exec/RegTests/SprayTest/input.2d#L91

Reply:
With these changes, there should not be a parcel_size input file entry. Instead, the parcel size is set based on how the particles are initialized. For jets, it is an input for the particular jet, and for uniform initialization the parcel size is set as an input here. https://github.com/AMReX-Combustion/PeleMP/blob/81ae98eeda7fd369cc12dcc57125a41be239aee9/Source/PP_Spray/SprayInjection.H#L207
The correct fix would be to have a problem specific input from the input file for the uniform spray cases that gets read in and passed through here.

Reply:
Ok. Will leave this alone for now because the cases can still run, but will look into adding this later